docs/brokerpak-intro.md

```diff
@@ -1,117 +1,65 @@
-# Creating Custom Services (UNRELEASED)
+# Brokerpak introduction
 
-This documentation is for an **UNRELEASED** upcoming feature for the GCP Service Broker and should not be considered complete.
+This document will explain `brokerpak`s, what they are, what problem they solve, how they solve that problem, and best practices around developing, maintaining, and using them.
```

Comment:
s/what problem they solve, how they solve that problem/what problem they solve, how they solve it